contracts/deploy-xtz-fa12.js

```diff
@@ -47,32 +47,32 @@ const estimateFees = async (store, web3) => {
         conseiljs.TezosParameterFormat.Micheline
       );
 
-    const xtzEstimate =
-      await conseiljs.TezosNodeWriter.testContractDeployOperation(
-        config.tezos.RPC,
-        config.tezos.chain_id,
-        store.keyStore,
-        0,
-        undefined,
-        100000,
-        10000,
-        20000,
-        trim(xtzConfig),
-        conseiljs.TezosLanguageUtil.translateMichelsonToMicheline(
-          `(Pair True (Pair "${store.keyStore.publicKeyHash}" {}))`
-        ),
-        conseiljs.TezosParameterFormat.Micheline
-      );
+    // const xtzEstimate =
```

Comment:
if this isn't needed anymore, please delete it. if it's needed under specific conditions, wrap it in an if